docs/release-1.2/existing-rhdh.md

```diff
@@ -31,10 +52,16 @@ In 1.2, the Orchestrator infrastructure is being installed using the orchestrato
         serviceNamespace: sonataflow-infra
       rhdhOperator:
         enabled: false
+
+      # Include if using OpenShift Gitops and OpenShift Piplines
+      tekton:
+        enabled: true
+      argocd:
+        enabled: true
     ```
 1. Verify resources and wait until they are running
-   1. From console run the following command get the necessary wait commands: \
-      `oc describe orchestrator orchestrator-sample -n openshift-operators | grep -A 10 "Run the following commands to wait until the services are ready:"`\
+   1. From the console run the following command in order to get the necessary wait commands: \
+      `oc describe orchestrator orchestrator-sample -n rhdh-operator | grep -A 10 "Run the following commands to wait until the services are ready:"`
```

Comment:
RHDH namespace is different than Orchestrator CR namespace.
we don't create resources for RHDH when installing on pre-existing RHDH

Reply:
so it shoudn't be `-n rhdh-operator`, or I'm misunderstanding?

Reply:
We don't enforce the namespace for this CR, but it should be the same value as the one mentioned in the Orchestrator CD definition in https://github.com/parodos-dev/orchestrator-helm-operator/pull/342/files#diff-3688824233dd051d9cce708de39906213b76573b52c8929522edf83e2ab6b1fbR33

perhaps we can just replace this value with a variable, e.g. `TARGET_NAMESPACE` or any other name to avoid hard-coding it.

Reply:
Ok I updated to $TARGET_NAMESPACE  - let me know if the change looks ok